### PR TITLE
Add date to Activity Feed for office pages. 

### PR DESCRIPTION
**Pull request by sebworks**
Add date to Activity Feed for office pages. 

## Changes
- Updated  '_includes/post-macros.html' to add optional date text
- Updated 'offices/_single.html'  to pass flag to display the date. 

## Test
- Visit 'http://localhost:7000/offices/*'

## ScreenShots
screen shot 2015-06-15 at 9 28 14 am

## Review 
- @jimmynotjim  
- @anselmbradford  
- @KimberlyMunoz 
- @kurtw

### CODE REVIEW

**Comment by KimberlyMunoz** [comment]
Great addition. It's one step closer to matching the mockups.

Can we include a class with `white-space: nowrap;` on the dates so they don't wrap onto two lines like on this one?

image

**Comment by sebworks** [comment]
Yep, will add that.

File: offices/_single.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
     {% from "post-macros.html" import activity_snippets as activity_snippets with context %}
     {% import "contact-macro.html" as contact_macro %}
-    {% set activities_feed = activity_snippets( office.tags if office.tags ) %}
+    {% set activities_feed = activity_snippets( office.tags if office.tags, include_date_flag=true) %}
```
Comment (KimberlyMunoz) [inline offices/_single.html]: Can you also include this change on `sub-page/_single.html`? We won't be able to test it until Content tags all those pages, but that should also use the same approach as the office-page.

**Comment by KimberlyMunoz** [comment]
:+1: I checked to make sure this didn't break on the office pages that we're presenting. I have a few suggestions above.

**Comment by sebworks** [comment]
ok, will do. Thanks @KimberlyMunoz

File: offices/_single.html (hunk 1 above)
Comment (sebworks) [inline offices/_single.html]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/e7de56fa93bd3faf587cc41a42b249154759f12d

File: offices/_single.html [hunk 2]
```diff
@@ -76,14 +89,18 @@ <h1 class="h3">{{ top_story.head | safe }}</h1>
     </section>
     {% endif %}
     {% if office.resources %}
-    <section class="office_resources block block__padded-top block__border-top">
+    <section class="office_resources
+                    block block__padded-top
```
Comment (jimmynotjim) [inline offices/_single.html]: should there be another break in here?

File: src/static/css/pages/offices.less [hunk 3]
```diff
@@ -32,6 +32,17 @@
             });
         }
     }
+
+    .activity {
```
Comment (jimmynotjim) [inline src/static/css/pages/offices.less]: Should these styles be moved out for all activity feeds (especially the date one)?

File: offices/_single.html (hunk 2 above)
Comment (sebworks) [inline offices/_single.html]: I'll add that.

File: src/static/css/pages/offices.less (hunk 3 above)
Comment (sebworks) [inline src/static/css/pages/offices.less]: yeah, probably should.

File: offices/_single.html (hunk 2 above)
Comment (sebworks) [inline offices/_single.html]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/e446711d95276e18e0a3d778a6040713215dd5c3

File: src/static/css/pages/offices.less (hunk 3 above)
Comment (sebworks) [inline src/static/css/pages/offices.less]: fixed in https://github.com/cfpb/cfgov-refresh/commit/e446711d95276e18e0a3d778a6040713215dd5c3

**Comment by jimmynotjim** [comment]
LGTM

**Comment by kurtw** [comment]
Sweet. I like it. :+1: Just fix conflicts.

**Pull request merged**